pallets/asset/src/tests.rs

```diff
@@ -226,6 +227,110 @@ fn asset_issue_should_succeed() {
 	});
 }
 
+#[test]
+fn asset_issue_should_succeed_until_limit() {
+    let creator = DID_00;
+    let author = ACCOUNT_00;
+    let capacity = 5u64;
+
+    let raw_space = [2u8; 256].to_vec();
+    let space_digest = <Test as frame_system::Config>::Hashing::hash(&raw_space.encode()[..]);
+    let space_id_digest = <Test as frame_system::Config>::Hashing::hash(
+        &[&space_digest.encode()[..], &creator.encode()[..]].concat()[..],
+    );
+    let space_id: SpaceIdOf = generate_space_id::<Test>(&space_id_digest);
+
+    let auth_digest = <Test as frame_system::Config>::Hashing::hash(
+        &[&space_id.encode()[..], &creator.encode()[..], &creator.encode()[..]].concat()[..],
+    );
+    let authorization_id: Ss58Identifier = generate_authorization_id::<Test>(&auth_digest);
+
+    let asset_desc = BoundedVec::try_from([72u8; 10].to_vec()).unwrap();
+    let asset_tag = BoundedVec::try_from([72u8; 10].to_vec()).unwrap();
+    let asset_meta = BoundedVec::try_from([72u8; 10].to_vec()).unwrap();
+    let asset_qty = 10;
+    let asset_value = 10;
+    let asset_type = AssetTypeOf::MF;
+
+    let entry = AssetInputEntryOf::<Test> {
+        asset_desc,
+        asset_qty,
+        asset_type,
+        asset_value,
+        asset_tag,
+        asset_meta,
+    };
+
+    let digest = <Test as frame_system::Config>::Hashing::hash(&[&entry.encode()[..]].concat()[..]);
+
+    new_test_ext().execute_with(|| {
+        let space_id_clone = space_id.clone();
+        let _authorization_id_clone = authorization_id.clone();
+
+        assert_ok!(Space::create(
+            DoubleOrigin(author.clone(), creator.clone()).into(),
+            space_digest,
+        ));
+
+        assert_ok!(Space::approve(RawOrigin::Root.into(), space_id_clone, capacity));
+
+        assert_ok!(Asset::create(
+            DoubleOrigin(author.clone(), creator.clone()).into(),
+            entry.clone(),
+            digest,
+            authorization_id.clone()
+        ));
+
+        for i in 1..=25 {
+            let _recipient = SubjectId(AccountId32::new([i as u8; 32])); 
+            let asset_id = space_id.clone(); 
```

Comment:
Why `asset_id` is set to `space_id`. 
There is a process to generate the `asset_id` is way that the chain understands/expects. Please refer other implementations.

Reply:
Resolved this